web/src/components/Strapi/Common/Card.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import Link from "next/link";
+import Markdown from "react-markdown";
+import { Button } from "@/types/strapi";
+
+type CardContentProps = {
+	headline: string;
+	text: string;
+	button?: Button;
+	hasShadow?: boolean;
+};
+
+export default function Card({ headline, text, button, hasShadow = false }: CardContentProps) {
+	return (
+		<div className={`${hasShadow && "flex flex-col rounded pt-7 pb-4 px-10 bg-gray-4 drop-shadow"}`}>
```

Comment:
this will result in `className=false` in the dom if `hasShadow` is false. Use a ternary operator for this please.

Either:
```suggestion
		<div className={`${hasShadow ? "flex flex-col rounded pt-7 pb-4 px-10 bg-gray-4 drop-shadow" : ""}`}>
```
Or (but please try this before committing the suggestion here):
```suggestion
		<div className={hasShadow ? "flex flex-col rounded pt-7 pb-4 px-10 bg-gray-4 drop-shadow" : undefined}>
```

Reply:
The second suggestion results in class=undefined
The first one in an empty class.
Could not observe any visible difference in the rendered result.
Implemented the first one.